src/OpenTelemetry.Instrumentation.EntityFrameworkCore/Implementation/EntityFrameworkDiagnosticListener.cs

```diff
@@ -180,6 +178,14 @@ public override void OnCustom(string name, Activity activity, object payload)
                                     break;
                             }
                         }
+                        try
+                        {
+                            this.options.EnrichWithIDbCommand?.Invoke(activity, command as IDbCommand);
+                        }
+                        catch (Exception ex)
+                        {
+                            EntityFrameworkInstrumentationEventSource.Log.EnrichmentException("EntityFrameworkCoreCommandCreated", ex);
```

Comment:
```suggestion
                            EntityFrameworkInstrumentationEventSource.Log.EnrichmentException(nameof(EntityFrameworkCoreCommandExecuting), ex);
```

Reply:
Yep, think I was just pushing the correction as you commented :D